HW#02 - Maslov Nikita

NikitaMasl opened the pull request.

etray7 commented:
It isn't responsive absolutely, you need to rework it.

etray7 commented:
![image](https://user-images.githubusercontent.com/26260209/72828259-56a60e80-3c8d-11ea-8130-dd07d0996430.png)

NikitaMasl commented:
I fixed flex and grid styles. I added new media request for normal display on large screens.
 
 
С уважением,
Маслов Н.Д.
 
  
>Вторник, 21 января 2020, 20:34 +03:00 от Kirill Litvinov <notifications@github.com>:
> 
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
 
